### PR TITLE
feat: add ecosystem file for pm2

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Puts out some fires, hopefully.

How to use:

```sh
cd api-server
pm2 start ecosystem.config.js
```

### CODE REVIEW

**Comment by ojeytonwilliams** [comment]
@raisedadead i assume the routing is set up to send requests to multiple ports, but if not this won't do a lot.

**Comment by ojeytonwilliams** [comment]
I think this is the way to go. It has the debug logging and clusters Just Work again.

**Comment by raisedadead** [comment]
Do you mind testing this later (once we all are done with a release) on staging?

File: api-server/ecosystem.config.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const dotenv = require('dotenv');
+
+const filePath = path.resolve('..', '.env');
+const env = dotenv.parse(fs.readFileSync(filePath));
+
+module.exports = {
+  apps: [
+    {
+      script: `./lib/production-start.js`,
+      env: { ...env, DEBUG: 'fcc*' },
```
Comment (raisedadead) [inline api-server/ecosystem.config.js:13]:
```suggestion
      env,
```

I really really really want to enable logs on demand.

**Review by raisedadead — CHANGES_REQUESTED** [review]

File: api-server/ecosystem.config.js (hunk 1 above)
Comment (ojeytonwilliams) [reply api-server/ecosystem.config.js:13]: Sure thing! Not sure what I was thinking there.

**Pull request merged**